src/libaktualizr/uptane/tuf.h

```diff
@@ -253,6 +257,7 @@ class Target {
   friend std::ostream &operator<<(std::ostream &os, const Target &t);
 
  private:
+  bool valid{true};
```

Comment:
`valid` as a boolean is slightly weird, but I think it's harmless for now and will be easy to change if we later need to.

Reply:
It's a bit weird to have inside the type I agree. Did you have a specific improvement idea?

Reply:
Not yet. I think it's fine for now.